pkg/controllers/installer/installer_controller_test.go

```
@@ -124,6 +125,32 @@ var _ = Describe("InstallerController", Serial, func() {
+			crName := "ccapio-testgadgets.test.example.com"
+
+			By("installing a revision with an unmanaged CRD")
+			revision := addRevisionWithUnmanagedCRDs(ctx,
+				[]string{providerMixed},
+				unmanagedCRDs,
+			)
+			setCompatibilityRequirementConditions(ctx, crName, true, true)
+			waitForRevision(ctx, revision.Name)
+
+			By("verifying the CompatibilityRequirement and ConfigMap both exist")
+			cr := &apiextensionsv1alpha1.CompatibilityRequirement{}
```

golangci-lint (2.12.2)

[error] missing whitespace above this line (invalid statement above assign)

(wsl_v5)